### ISSUE TITLE
the results

### ISSUE THREAD

**Issue by ervinyo** [S-Choudhuri/Adversarial-Domain-Adaptation-with-Keras#2]
Thank you for sher this code, but i just confuse how to choose the better model ?
for example, I am getting this results

iter: 02504: 
LABEL CLASSIFICATION: source_accuracy: 100.00000, target_accuracy: 64.00000                    
DOMAIN DISCRIMINATION: source_domain_accuracy: 74.00000, target_domain_accuracy: 40.00000 

iter: 03005: 
LABEL CLASSIFICATION: source_accuracy: 100.00000, target_accuracy: 44.00000                    
DOMAIN DISCRIMINATION: source_domain_accuracy: 88.00000, target_domain_accuracy: 0.00000 

iter: 04508: 
LABEL CLASSIFICATION: source_accuracy: 100.00000, target_accuracy: 36.00000                    
DOMAIN DISCRIMINATION: source_domain_accuracy: 20.00000, target_domain_accuracy: 8.00000 

what is the mean ? please reply as soon as possible. Thank you

**Comment by S-Choudhuri** [comment]
For classification, the essential accuracy value is the "target_accuracy". In your case, I guess the model is poorly generalizing on the target dataset. In that case, use data augmentation during model training (the data augmentation code needs to be written separately) and tweak the hyper-parameters of the model accordingly.

**Comment by ervinyo** [comment]
> For classification, the essential accuracy value is the "target_accuracy". In your case, I guess the model is poorly generalizing on the target dataset. In that case, use data augmentation during model training (the data augmentation code needs to be written separately) and tweak the hyper-parameters of the model accordingly.

Thank you for your reply but I confuse the different between label classification and domain discrimination since you are using ADDA, focus in the discrimination accuracy.
Can you explain to me the different about label classification and domain discrimination ?

**Comment by S-Choudhuri** [comment]
label classification accuracy caters to the proper categorization of images in each domain (for example, classifies an image into labels like: camera, table, etc). domain discrimination accuracy on the other hand, aids in discriminating whether an image belong to the source domain or the target domain. The discriminator in the model takes care of the domain discrimination, while the classifier performs the label classification of images.

**Comment by ervinyo** [comment]
> label classification accuracy caters to the proper categorization of images in each domain (for example, classifies an image into labels like: camera, table, etc). domain discrimination accuracy on the other hand, aids in discriminating whether an image belong to the source domain or the target domain. The discriminator in the model takes care of the domain discrimination, while the classifier performs the label classification of images.

I see. Thank you
